packages/datagrid/src/components/HeaderCellRenderer/HeaderCellRenderer.scss

```diff
@@ -0,0 +1,89 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+$td-grip-resize-width: 6px !default;
+
+:global(.ag-header-cell) {
+	:global(.ag-header-cell-resize) {
+		width: $td-grip-resize-width;
+	}
+}
+
+@mixin ellipsis {
+	white-space: nowrap;
+	text-overflow: ellipsis;
+	overflow: hidden;
+}
+
+.header-cell {
+	height: 100%;
+	width: 100%;
+	display: flex;
+	flex-direction: column;
+	padding: tokens.$coral-spacing-xs;
+
+	:global(.tc-ratio-bar) {
+		padding: tokens.$coral-spacing-xs 0 0;
+	}
+
+	&__button {
+		display: flex;
+		align-items: center;
+		justify-content: space-between;
+		padding: 0;
+		text-align: left;
+		background-color: transparent;
+		border: none;
+		gap: tokens.$coral-spacing-xxs;
```

Comment:
Stack component can be used here, but you need the wrapper to bind some styles/behavior so... up to you

Reply:
good catch, this div used to be a button element, I forgot to clean up the CSS